functions.php

```diff
@@ -436,7 +433,13 @@ function twentytwenty_add_classic_editor_customizer_styles( $mce_init ) {
 	function twentytwenty_block_editor_settings() {
 
 		// Block Editor Palette.
-		$editor_color_palette = array();
+		$editor_color_palette = array(
+			array(
+				'name'  => esc_html__( 'Accent Color' ),
+				'slug'  => 'accent',
+				'color' => twentytwenty_get_color_for_area( 'content', 'accent' ),
+			),
+		);
 
```

Comment:
The editor color palette should include all 6 of the colors used in the theme that I noted in #284. The custom colors script should define the modified versions of all six colors (where black and white likely won't be modified or redefined here, but multiple accent colors could be). This is another place where we can't rely on opacity for the secondary and border colors, but need to calculate an actual color.

Reply:
I have no problem adding as many calculations and variables as we want.
However, someone will have to tell me how to get these colors.
I know how to get the color for text & accent, but what about the others? How are they calculated? Do we rotate the hue X degrees, then get a color that's 10% the rotated hue, with a saturation Y and a lightness that is 80 background - 20% text?

Reply:
Added the extra colors to the palette